client/components/datapicker/datapicker.html

```diff
@@ -0,0 +1,38 @@
+<link rel="stylesheet" href="datapicker.css" />
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.0/css/all.min.css" />
+<title>Data picker</title>
+</head>
+<body>
+  <div class="datepicker">
+    <button name="buttonSubmit" id="buttonSubmitId" class="datepicker-button" onclick="ShowMyDatepicker()"></button>
```

Comment:
Let's use `.addEventListener("click", () => ShowMyDatepicker())` instead? Like you did for some elements in the `datapicker.js`.